src/app/onprem/license-nag-message-management/nag-message.service.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2015-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ */
+'use strict';
+
+
+/**
+* This class is handling the service for the nag message
+* @author Oleksii Orel
+*/
+export class NagMessageService {
+
+  /**
+   * Default constructor that is using resource injection
+   * @ngInject for Dependency injection
+   */
+  constructor($document, cheBodyInjectorSvc , codenvyLicense) {
+    'ngInject';
+
+    this.$document = $document;
+    this.cheBodyInjectorSvc = cheBodyInjectorSvc;
+    this.codenvyLicense = codenvyLicense;
+
+  }
+
+  /**
+   * Show nag message
+   */
+  showNagMessage() {
+    this.$document.find('body').addClass('nag-message-indent');
```

Comment:
so you may add addStyle() on cheBodyInjectorSvc ?
also it's not possible to manage the style only in what we had in the body ?
usually it should shift the whole app by inserting an element at the top of the body element, before any other elements